src/main/java/kata/makers/service/FlightService.java

```diff
@@ -0,0 +1,56 @@
+package kata.makers.service;
+
+import java.util.Map;
+import java.util.Optional;
+import java.util.Scanner;
+import kata.makers.exception.CreateFlightException;
+import kata.makers.model.Flight;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+
+public class FlightService {
```

Comment:
Could you add empty lines here between class name and variables, and then variables and the constructor?